Docs k8smon whats new content (#76153)
grafana/grafana: docs/sources/whatsnew/whats-new-next/index.md

Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.

## Quicker path to troubleshooting in Kubernetes Monitoring

<!-- Beverly Buchanan --> 

_Generally available in Grafana Cloud_

> You can use Kubernetes Monitoring to more quickly identify and troubleshoot issues with these views:

Reviewer: ```suggestion
You can use Kubernetes Monitoring to more quickly identify and troubleshoot issues with these added views:
```

- The home page shows a [snapshot of issues](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/navigate-k8s-monitoring/#see-the-issues-at-a-glance) that reach a specific threshold. 
  
  (ADD SCREENSHOT AFTER CROW's NEST LANGUAGE REMOVED)

  This view shows:

  - Pods that have been in a non-running state for more than 15 minutes
  - Nodes with CPU usage above 90% for more than five minutes
  - Nodes using more than 90% of memory for more than five minutes

>   - Persistent Volumes with capacity above 90%

Reviewer: I assume "Volumes" is intentionally capitalized here?

Author: Yes

  - Node disks with capacity above 90%

- The [**Cluster Navigation** page](docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/navigate-k8s-monitoring/#drill-down-into-data/) provides tabs that allow you to quickly switch between the Cluster, namespace, workload, and Node views.

  (ADD SCREENSHOT)

## Configuration additions in Kubernetes Monitoring

<!-- Beverly Buchanan --> 

> <!-- Cloud --> 

Reviewer: Don't need this bit. Only need to add something here if the feature is available in OSS and/or Enterprise.
```suggestion
```

_Generally available in Grafana Cloud_

> You have these options when configuring Kubernetes Monitoring:

Reviewer: ```suggestion
You have two new options when configuring Kubernetes Monitoring:
```

> - Grafana Kubernetes Monitoring Helm chart: To send infrastructure data to Grafana Cloud, you can configure easily using the Helm chart. With this method, you can select toggle buttons to gather metrics, logs, events, traces, and cost metrics.

Reviewer: "configure easily" without a thing you're configuring as part of the phrase sounds odd. I don't have a suggestion because I'm not sure what the missing piece is. Also if we can avoid the repetition of "you can" in both sentences, that would be better.

Author: Rewrote a lot of content. Good point.

(NEED TO ADD SCREENSHOT HERE)

- Traces: You can collect traces when you configure Kubernetes Monitoring, and then use Tempo to create search queries. Refer to [Navigate to traces](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/navigate-k8s-monitoring/#navigate-to-traces) for more information.

## Cost monitoring in Kubernetes Monitoring

<!-- Beverly Buchanan --> 

> <!-- Cloud --> 

Reviewer: ```suggestion
```

_Generally available in Grafana Cloud_

> You can use the Cost view to discover insights about spending, and make data-driven decisions about resource allocation, scaling, and tech investments. This feature is based on an [OpenCost](https://www.opencost.io/) integration and Grafana’s experience in managing Kubernetes-related costs. For more information, refer to [Manage costs](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/manage-costs/).

Reviewer: ```suggestion
You can use the Cost view to discover insights about spending and make data-driven decisions about resource allocation, scaling, and tech investments. This feature is based on an [OpenCost](https://www.opencost.io/) integration and Grafana’s experience in managing Kubernetes-related costs. For more information, refer to [Manage costs](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/manage-costs/).
```

Author: This content was removed because it was older upgrade.

Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.

> ## User experience updates in Kubernetes Monitoring

Reviewer: So we haven't been grouping these by area/product because the idea for the next iteration of the Cloud what's new is that they'd be sortable by date or any number of other tags. "Ungrouping these" would entail:

- Removing the current H2 heading and text
- Making all the H3s into H2s
- Updating the titles to make clear that they relate to K8s
- Adding the contributor and availability information to each of entry 

I don't want to create extra work for you, but I don't know how this will translate when we switch to a new system in November/December. So I'll leave it up to you how you want to proceed.

Author: Interesting. Within K8s Mon are various releases, plus the plugin version. I'll talk to dev about this today. If we ungroup things, the things still show up in the same GUI. Will that be confusing to users? Or tedious because of the repeated "Kubernetes Monitoring" in the heading? I'm happy to do whatever works.

<!-- Beverly Buchanan --> 

> 

Reviewer: We want to add a release date for all entries. If you keep these grouped together then just do them in the two spots where you've grouped them. If not, then you'll want to add them to to each entry after the heading and before the availability note with a space on either side.
```suggestion

<Date>

```

_Generally available in Grafana Cloud_

> ### Quickly troubleshoot with snapshot of issues

Reviewer: ```suggestion
### Quickly troubleshoot with a snapshot of issues
```

> Release 1.3.1. To provide quicker identification and troubleshooting, you can view a snapshot of issues on the home page that reach these specific thresholds:

Reviewer: Are you viewing:
- a snapshot of issues that reach specific thresholds and that view is on the home page OR
- a snapshot of issues that are on the home page + reach specific thresholds?

It's not clear which you mean, so I would recast the sentence to clarify

Author: Thanks for the feedback. I'll work on it!

  - Pods that have been in a non-running state for more than 15 minutes
  - Nodes with CPU usage above 90% for more than five minutes
  - Nodes using more than 90% of memory for more than five minutes
  - Persistent Volumes with capacity above 90%
  - Node disks with capacity above 90%

![Kubernetes Monitoring home page](/media/docs/grafana-cloud/k8s/K8smon-snapshotview.png)

### Navigate with tabs

> Release 1.3.0. Quickly switch between the Cluster, namespace, workload, and Node views on the **Cluster Navigation** page by using tabs.

Reviewer: ```suggestion
Release 1.3.0. Quickly switch between the Cluster, namespace, workload, and Node views on the **Cluster Navigation** page using tabs.
```

> ![Tabs on Cluster navigation page](/media/docs/grafana-cloud/k8s/k8smon-clusternav-tabs.png)

Reviewer: This image is really big for what it's showing. Suggest using a `[figure` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#example-5) to constrain its width (see second example at this link).

Author: Thx! Resized

### Change data source on Cost and Efficiency views

Release 1.2.1. You can change the data source you are viewing on the Cost and Efficiency views. 

### Predict namespace memory usage

Release 1.2.1. Click **Predict Memory usage** to predict namespace memory usage on the namespace detail page.

![Predict Memory usage button](/media/docs/grafana-cloud/k8s/k8smon-predict-memusage-namespace.png)

## Configuration updates in Kubernetes Monitoring

<!-- Beverly Buchanan --> 

_Generally available in Grafana Cloud_

### Configure with streamlined process

> Release 1.2.0. Configure with [Grafana Kubernetes Monitoring Helm chart]({{< relref "/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/config-k8s-agent-flow" >}}) using a streamlined process. With this method, you select with toggle buttons to gather metrics, logs, events, traces, and cost metrics.

Reviewer: We get to use simple full URLs now! However, this URL did a 404 when I tested it locally.
```suggestion
Release 1.2.0. Configure with [Grafana Kubernetes Monitoring Helm chart](https://grafana.com/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/config-k8s-agent-flow) using a streamlined process. With this method, you select with switches to gather metrics, logs, events, traces, and cost metrics.
```

> ![Streamlined configuration wizard](/media/docs/grafana-cloud/k8s/k8smon-config-wizard.png)

Reviewer: I'd consider constraining this image as well
